posthog/clickhouse/cluster.py

```diff
@@ -27,7 +28,7 @@
 from posthog.settings.data_stores import CLICKHOUSE_CLUSTER
 
 
-logger = logging.getLogger(__name__)
+logger = dagster.get_dagster_logger("clickhouse")
```

Comment:
This is a hack - I couldn't track down a good way to programmatically get these logs to show up in the Dagster UI (which I think is important) without using this helper which basically puts the logger under the `dagster.*` namespace

Unfortunately this means that other non-Dagster related logs (like during migrations that use the cluster client) will show up under that namespace for now which is a bit misleading, but I don't think there's a good way to alter that behavior at runtime other than passing the logger down the stack everywhere it's used, which is annoying

More on this in the next comment